tests/bpf2c_tests/expected/test_utility_helpers_sys.c

```diff
@@ -173,222 +173,267 @@ division_by_zero(uint32_t address)
 
 #include "bpf2c.h"
 
-static void _get_hash(_Outptr_result_buffer_maybenull_(*size) const uint8_t** hash, _Out_ size_t* size)
+static void
+_get_hash(_Outptr_result_buffer_maybenull_(*size) const uint8_t** hash, _Out_ size_t* size)
 {
-	*hash = NULL;
-	*size = 0;
+    *hash = NULL;
+    *size = 0;
 }
 static map_entry_t _maps[] = {
-{ NULL, { 2, 4, 32, 2, 0, 0, 0, 0,  }, "utility_map" },
+    {NULL,
+     {
+         2,
+         4,
+         32,
+         2,
+         0,
+         0,
+         0,
+         0,
```

Comment:
Can we add // comments on each of these lines saying what the number is?

Reply:
Good idea:
https://github.com/microsoft/ebpf-for-windows/issues/1063